tests/components/image_processing/test_facebox.py

```diff
@@ -126,6 +141,61 @@ def mock_face_event(event):
             PARSED_FACES[0][fb.ATTR_BOUNDING_BOX])
 
 
+@patch('os.access', Mock(return_value=True))
+@patch('os.path.isfile', Mock(return_value=True))
```

Comment:
There's a problem with this test. It's never awaited. Unfortunately that passes silently. I think it's because of these patch decorators.

I noticed if after checking the coverage report.